api/mysagw/conftest.py

```diff
@@ -24,15 +24,16 @@
 TEST_FILES_DIR = Path(__file__).parent.resolve() / "test_files"
 
 
-def register_module(module):
-    for name, obj in inspect.getmembers(module):
-        if isinstance(obj, FactoryMetaClass) and not obj._meta.abstract:
-            register(obj)
-
-
-register_module(importlib.import_module(".identity.factories", "mysagw"))
-register_module(importlib.import_module(".snippets.factories", "mysagw"))
-register_module(importlib.import_module(".case.factories", "mysagw"))
+register(identity_factories.IdentityFactory)
+register(identity_factories.EmailFactory)
+register(identity_factories.PhoneNumberFactory)
+register(identity_factories.AddressFactory)
+register(identity_factories.InterestCategoryFactory)
+register(identity_factories.InterestFactory)
+register(identity_factories.MembershipRoleFactory)
+register(identity_factories.MembershipFactory)
+register(snippets_factories.SnippetFactory)
+register(case_factories.CaseAccessFactory)
```

Comment:
At this point, I think you could just use `register()` as a decorator in the factory module as it was originally intended, no?

Reply:
tbh I tried (very briefly), but they were not available then. Also tried with an import from conftest. Thing is, we want those factories to be available globally. The examples in the docs have the factory in the same file as the test.

Reply:
Ok, I've tried it again (with the right imports this time ;)) and it does work. But looking at it, I think I prefer the solution with the `register()` calls in `conftest.py`, because it makes it obvious, that we're registering the Factories as opposed to the unused imports of the factories, that implicitly adds them to the scope of the `conftest.py`.